src/server/metrics.rs

```diff
@@ -34,6 +34,19 @@ lazy_static! {
             &["type"]
         ).unwrap();
 
+    pub static ref GRPC_MSG_FAIL_COUNTER: CounterVec =
+        register_counter_vec!(
+            "tikv_grpc_msg_fail_total",
```

Comment:
I think the name should contain _grpc_ or _msg_ at least, _process_ is too ambiguous.